packages/stylelint-config-terra/stylelint.config.js

```diff
@@ -0,0 +1,48 @@
+const browserslist = require('@cerner/browserslist-config-terra');
+
+module.exports = {
+  extends: 'stylelint-config-sass-guidelines',
+  plugins: [
+    'stylelint-no-unsupported-browser-features',
+    './lib/rules/custom-property-no-duplicate-declaration/custom-property-no-duplicate-declaration',
+    './lib/rules/custom-property-name/custom-property-name',
+    './lib/rules/custom-property-pattern/custom-property-pattern',
+    './lib/rules/custom-property-pseudo-selectors/custom-property-pseudo-selectors',
+    './lib/rules/custom-property-namespace/custom-property-namespace',
+  ],
+  rules: {
+    'max-nesting-depth': 3,
+    'no-extra-semicolons': [true, { severity: 'warning' }],
+    'scss/at-mixin-pattern': '^[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
+    'custom-property-pattern': [
+      '[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
```

Comment:
Its been a while since I've looked at these custom rules, but do we need to add the custom-property-pattern if we have created out own custom rule fo this that is using similar regex? 

- [regex ref](https://github.com/cerner/terra-toolkit/pull/504/files#diff-504e4391133e20ce330bdf9151d6dd53065993182a2c26d41c64eec830e4c007R12)

@StephenEsser

Reply:
My memory is a little fuzzy on this rule but I think we need to keep it for now. I think our custom rule lints custom properties defined with the `var` syntax and this run lints by declaration. 

Example: These are linted by different rules. 

```scss
.variables {
  --example-color-red: red;
 
  background-color: var(red, --example-color);
}
```